webapp/components/wpt-results.html

```diff
@@ -243,7 +243,8 @@
         /* Recomputes the list of displayed directories and test files. */
         const displayedNodeMap = new Map();
         const currentPathParts = this.splitPathIntoLinkedParts(this.path);
-
+        const partCountIfTestFile = this.path == '/' ? 1 : currentPathParts.length + 1;
+        const nextSubdirIndex = this.path == '/' ? 0 : currentPathParts.length;
```

Comment:
This is always the above - 1. Incidental, or can one be defined in terms of the other?